Tools/ardupilotwaf/build_summary.py

```diff
@@ -154,7 +154,7 @@ def _build_summary(bld):
             Logs.info('')
             Logs.pprint(
                 'NORMAL',
-                'Note: Some targets were suppressed. Use --summary-all if you want information of all targets.',
+                '\033[0;31;1mNote: Some targets were suppressed. Use --summary-all if you want information of all targets.',
```

Comment:
I am pretty sure this won't work on all terminal. Can't we use a waf function to have the right color ?

Reply:
waf uses ANSI too I think, https://github.com/ArduPilot/waf/blob/51f1529bd20759642e9fd0a8db648da9ba380b18/configure#L14